RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java

```diff
@@ -45,12 +45,12 @@ private Command getSingleBallShootCommand()
     private Command getDoubleBallShootCommand()
     {
         return new InstantCommand(this.top ? () -> this.shooter.spinUpTop() : () -> this.shooter.shootLow()) //Activate shooter
-                .andThen(new WaitUntilCommand(() -> this.shooter.isUpToSpeed()) //Wait until shooter is up to speed
+                .andThen(new WaitUntilCommand(() -> this.shooter.isUpToHighSpeed()) //Wait until shooter is up to speed
```

Comment:
Can you fix this to work with low shots as well? See the ternary operator above.